react/components/EditorContainer/StoreEditor/Styles/StyleEditor/typography/FileFontEditor.tsx

```diff
@@ -96,7 +117,7 @@ const FileFontEditor: React.FunctionComponent<FileFontEditorProps> = ({
   )
 }
 
-interface FontFileItemProps {
+interface FontFileItemProps extends Pick<InjectedIntlProps, 'intl'> {
```

Comment:
Is this `Pick` necessary? I think there's only `intl` inside `InjectedIntlProps` already:
![Screen Shot 2019-07-01 at 18 10 38](https://user-images.githubusercontent.com/10400340/60466954-8b884080-9c2b-11e9-9d1d-6411b923153a.png)

Reply:
Good one, mixed it up with `RouteComponentProps` :(